opentelemetry/src/global/internal_logging.rs

```diff
@@ -0,0 +1,130 @@
+#![allow(unused_macros)]
+/// Macro for logging messages at the general log level in OpenTelemetry.
```

Comment:
a warning at the top to indicate that this meant for user by OpenTelemetry components itself.